src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -871,6 +821,23 @@ internal static void BlockingEndWrite(IAsyncResult asyncResult)
         {
             SynchronousAsyncResult.EndWrite(asyncResult);
         }
+        
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
@justinvp Doing a GitHub search for "ContractArgumentValidator" in this repo doesn't turn up anything, so I'm guessing no as well. @jkotas let me know otherwise.